src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$grey-color: #616070;
+$dark-blue-color: #060b35;
+$blue-color: #00acdc;
+$product-card-width: 200px;
+$product-card-side-margins: 48px;
+$catalog-width: $product-card-width * 4 + $product-card-side-margins * 3;
+$star-measures: 16px;
```

Comment:
```suggestion
$star-size: 16px;
```

Reply:
I didn't want to give the same name for variable and mixin. But ok